Networking/Networking/Model/Order.swift

```diff
@@ -162,8 +162,8 @@ public struct Order: Decodable, GeneratedCopiable, GeneratedFakeable {
         let currencySymbol = try container.decode(String.self, forKey: .currencySymbol)
         let customerNote = try container.decode(String.self, forKey: .customerNote)
 
-        let dateCreated = try container.decodeIfPresent(Date.self, forKey: .dateCreated) ?? Date()
-        let dateModified = try container.decodeIfPresent(Date.self, forKey: .dateModified) ?? Date()
+        let dateCreated = (try? container.decodeIfPresent(Date.self, forKey: .dateCreated)) ?? Date()
+        let dateModified = (try? container.decodeIfPresent(Date.self, forKey: .dateModified)) ?? Date()
```

Comment:
I know that these already had a nill fallback but I wonder if it is ideal to list orders with a different order than the intended one, and how could this affect merchant workflows 🤔

Reply:
Yeah, this was the one fallback I was most skeptical about. But given that the other order data is the same (including status), the orders could still be processed in the same way — just, as you mentioned, maybe a different order. I think that is still better than failing to decode the order list entirely. We can watch out for any feedback about it, though, and a future improvement would still be to show some kind of message in the UI if we have an issue with the decoding.